docs/README.md

```diff
@@ -19,13 +19,12 @@ Once you have `sphinx` installed, you can build the documentation and enable wat
 » make watch-docs
 ```
 
-For Apple Silicon (M1), a environment variable is required:
-```bash
-» export PYENCHANT_LIBRARY_PATH=/opt/homebrew/lib/libenchant-2.2.dylib
-```
-then install pychant with `brew`:
```

Comment:
We should still keep this solution here in case users have a different dylib location for libenchant.

I will add a follow up into how to setup this with nix

Reply:
We can add it back. TBH, not totally sure how consistent this fix will be, that's why i added the link to the main pyenchant site. That solution seemed to work.

When brew installed enchant, I think it set this path automatically for me

Reply:
that leaves you with having to manage two brew instance. I think it still has some value to let users know that this options also exist.